clients/upsmon.c

```diff
@@ -657,6 +657,10 @@ static int is_ups_critical(utype_t *ups)
 
 	/* must be OB+LB now */
 
+	/* if UPS is calibrating, don't declare it critical */
+	if (flag_isset(ups->status, ST_CAL))
+		return 0;
```

Comment:
Would it make sense to log this situation, especially to help troubleshoot when/if this backfires (e.g. we had an authentic outage during calibration, really ran out of power and crashed ungracefully - and here is the log message why NUT did not shut down the system).

Are there more factors to check (e.g. does an UPS have a bypass, is it live or un-powered during the calibration)?

Reply:
Added the logging bit myself :)